### PR TITLE
KR table - critical commercial assays + TCGA

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose
❗ staggered on #222 

Adding critical commercial assays + TCGA source information + cell line info to the KR table 

### Issue

Not a current issue, but I filled this in while I await a response for #217 

#### Which areas should receive a particularly close look?
- Just copied over relevant info from the methods 

 
#### Is there anything that you want to discuss further?
- Also made minor updates within methods to be consistent with the table

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/06.methods.md [hunk 1]
```diff
@@ -588,34 +588,46 @@ EXTEND scores have been further compared, using Spearman rank correlation, betwe
...
 |Critical commercial assays|||
-|                    |                        |                    |
-|                    |                        |                    |
-|                    |                        |                    |
-|                    |                        |                    |
+| Recover Cell Culture Freezing media    | Gibco           | 12648010    |
+| Hank's Balanced Salt Solution (HBSS)   | Gibco           | 14175095    |
+| Papain                                 | SciQuest        | LS003124    |
+| Ovomucoid                              | SciQuest        | 542000      |
+| DNase                                  | Sigma           | 10104159001 |
```
Comment (sjspielman) [inline content/06.methods.md:596]: looks like Roche https://www.sigmaaldrich.com/US/en/product/roche/10104159001

File: content/06.methods.md [hunk 2]
```diff
@@ -588,34 +588,46 @@ EXTEND scores have been further compared, using Spearman rank correlation, betwe
...
+| Epidermal growth factor                | Gibco           | PHG0311L    |
+| Basic fibroblast growth factor         | PeproTech       | 100-18B     |
+| Heparin                                | Sigma           | H3149       |
+| DNA/RNA AllPrep Kit                    | Qiagen          | 80204       |
+| TruSeq RNA Sample Prep Kit             | Illumina        | FC-122-1001 |
+| KAPA Library Preparation Kit           | Roche           | KK8201      |
+| AllPrep DNA/RNA/miRNA Universal kit    | Qiagen          | 80224       |
+| RNase A                                | Qiagen          | 19101       |
+| QIAsymphony DSP DNA Midi Kit           | Qiagen          | 937255      |
+| KAPA HyperPrep kit                     | Roche           | KK8541      |
```
Comment (sjspielman) [inline content/06.methods.md:614]: Double check `KK8541`
https://sequencing.roche.com/en/products-solutions/products/sample-preparation/dna-reagents/library-preparation/kapa-hyperprep/ordering.html

Comment (sjspielman) [reply content/06.methods.md:614]: See: https://rochesequencingstore.com/?s=KK8541&post_type=product&type_aws=true

File: content/06.methods.md [hunk 3]
```diff
@@ -588,34 +588,46 @@ EXTEND scores have been further compared, using Spearman rank correlation, betwe
...
+| RiboErase kit                          | Roche           | KK8484      |
```
Comment (sjspielman) [inline content/06.methods.md:615]: Double check `KK8484` - https://rochesequencingstore.com/?s=KK8484&post_type=product&type_aws=true

Comment (jharenza) [reply content/06.methods.md:615]: I will have to check with @mkoptyra on the rest of these - I wonder if they used to exist and now they don't or if they are wrong.

Comment (jharenza) [reply content/06.methods.md:615]: hmm, looks like roche has a number different from kapa - will have to change

Comment (jharenza) [reply content/06.methods.md:615]: updated to 07962304001

**Comment by mkoptyra** [comment]
@jharenza I don't think it matters but KK numbers from Kapa are still in use/referenced, so either one is correct

**Pull request merged**